### PR TITLE
removed redundant checks from rational.h

### PR DESCRIPTION
**Pull request by abhinavagarwal07**
Solved #693

### CODE REVIEW

**Comment by rwst** [comment]
+1

File: symengine/rational.h [hunk 1]
```diff
@@ -55,12 +55,12 @@ class Rational : public Number {
     inline bool is_int() const { return this->i.get_den() == 1; }
```
Comment (isuruf) [inline symengine/rational.h]: This method is not needed right?

**Comment by isuruf** [comment]
+1. Can you also fix the other methods like `is_one`?

**Comment by abhinavagarwal07** [comment]
sure but these should these functions return false or removed?

**Comment by isuruf** [comment]
They should return false and `is_int` is not really necessary.

**Comment by abhinavagarwal07** [comment]
anything else?

**Comment by certik** [comment]
That looks good to me, thanks!

**Pull request merged**